tests/adapter/dbt/tests/adapter/simple_snapshot/test_snapshot.py

```diff
@@ -104,7 +104,8 @@ def _assert_results(
             {(i, True) for i in ids_with_current_snapshot_records},
             {(i, False) for i in ids_with_closed_out_snapshot_records},
         )
-        assert records == expected_records
+        for record in records:
```

Comment:
do we know what makes the two objects unalike for snowflake?

Reply:
it was matching sets where are unordered. the equality breaks because it expects a set to be in a particular order. The record and expected_records have the same items but the ordering is not deterministic (as expected in a set)

Reply:
oh neat, thank you for the detailed explanation!.